Xamarin.Essentials/Clipboard/Clipboard.netstandard.tvos.watchos.tizen.cs

```diff
@@ -4,6 +4,31 @@ namespace Xamarin.Essentials
 {
     public static partial class Clipboard
     {
+#if GTK
```

Comment:
@webwarrior-ws but you're copying code from Xamarin.Essentials/Clipboard/Clipboard.gtk.cs file, therefore it seems that file has to be deleted in this same commit, right?

Reply:
I was copying from Clipboard.netstandard.tvos.watchos.tizen.cs from stable branch, but you are right

Reply:
ok, let's then delete that file and squash the deletion into the Clipboard commit